src/components/toot/index.js

```diff
@@ -1,27 +1,34 @@
 import React, { Component } from 'react'
 import PropTypes from 'prop-types'
+import { translate } from 'react-i18next'
 
 import { avatarUrl } from '../../utils/parser'
 import './styles.css'
 
 class Toot extends Component {
 	render() {
-		const { account, reblog, content, created_at } = this.props
-		const tootContent = (reblog && reblog.content) || content
-
-		const avatar = avatarUrl(account)
+		const { t } = this.props
+		const isReblog = this.props.reblog && this.props.reblog.content
+		const toot = isReblog ? this.props.reblog : this.props
+		const avatar = avatarUrl(toot.account)
```

Comment:
Sorry, I just meant a component for the avatar. 
It could receive a toot and deal with finding who's the author and display the correct image.

(But it also means that it would receive more data than needed... Not sure about this :D)